caffe2/utils/math_cpu.cc

```diff
@@ -734,6 +736,15 @@ DELEGATE_COSH_FUNCTION(float)
 DELEGATE_COSH_FUNCTION(double)
 #undef DELEGATE_COSH_FUNCTION
 
+#define DELEGATE_INV_FUNCTION(T)                                        \
+  template <>                                                           \
+  void Inv<T, CPUContext>(const int N, const T* x, T* y, CPUContext*) { \
```

Comment:
Fix lint? the '\' should be in the same column

Reply:
''?